pkgs/tools/misc/massren/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, pkgs }:
```

Comment:
Please remove the 'pkgs' argument. (It's both unused and not recommended to take it as argument -- better be explicit about dependencies.)

Reply:
@bjornfor - Fixed and force-pushed!